src/Uecode/Bundle/ApiKeyBundle/Security/Firewall/ApiKeyListener.php

```diff
@@ -17,7 +17,7 @@
 class ApiKeyListener implements ListenerInterface
 {
     /**
-     * @var SecurityContextInterface
+     * @var TokenStorageInterface
      */
     protected $securityContext;
```

Comment:
Yeah, not a bad idea. I can do a major release too.

Aaron Scherer




On Thu, Jan 14, 2016 at 8:06 AM -0800, "Gennady Telegin" <notifications@github.com> wrote:












In src/Uecode/Bundle/ApiKeyBundle/Security/Firewall/ApiKeyListener.php:
> @@ -17,7 +17,7 @@
>  class ApiKeyListener implements ListenerInterface
>  {
>      /**
> -     * @var SecurityContextInterface
> +     * @var TokenStorageInterface
>       */
>      protected $securityContext;



done, but is it safe for BC? Field is protected so it can be used by class childs.

May be make these fields private?



—
Reply to this email directly or view it on GitHub.